.github/pull_request_template.md

```diff
@@ -1,46 +1,87 @@
-## Cover letter
+<!--
+
+See https://github.com/redpanda-data/redpanda/blob/dev/CONTRIBUTING.md##pull-request-body
+for more details and examples of what is expected in a PR body.
+
+-->
 
-Describe in plain language the motivation (bug, feature, etc.) behind the change in this PR and how the included commits address it.
+## Cover Letter
```

Comment:
I wonder if we should remove the ## Cover Letter heading entirely, and take is as read that the PR description is a cover letter?  It bugs me a little because if I open a PR from one commit, the commit message that's the default PR description gets pasted in above the heading, so moving the heading is the first thing I end up doing.

Reply:
+1 on dropping `## cover letter`. i agree it's redundant.

Reply:
please

Reply:
done